src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,38 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final String pattern = "dd.MM.yyyy";
+    private static final DateTimeFormatter SIMPLE_DATE_FORMAT =
+            DateTimeFormatter.ofPattern(pattern);
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int moneyEarned;
+        String[] dataOfIndex;
+        LocalDate dateFrom2 = LocalDate.parse(dateFrom, SIMPLE_DATE_FORMAT).minusDays(1);
+        LocalDate dateTo2 = LocalDate.parse(dateTo, SIMPLE_DATE_FORMAT).plusDays(1);
+        LocalDate dateOfWork;
+        StringBuilder reportForPeriod = new StringBuilder()
+                .append("Report for period ").append(dateFrom)
```

Comment:
```suggestion
                .append("Report for period ")
                .append(dateFrom)
```

Reply:
all fixed 
Can't change status to "r" on the table to